idseq_dag/steps/run_star.py

```diff
@@ -152,11 +157,13 @@ def sync_pairs_work(of0, of1, if0, if1):
         outstanding_r1 = {}
         mem = 0
         max_mem = 0
+        total = 0
         while True:
             r0, r0id = PipelineStepRunStar.get_read(if0)
             r1, r1id = PipelineStepRunStar.get_read(if1)
             if not r0 and not r1:
                 break
+            total += 1
```

Comment:
`total += 2`

Reply:
Are we not counting PAIRS here? see https://github.com/chanzuckerberg/idseq-dag/blob/074f7eef2dfe962017c3df70c85d3739865e1f62/idseq_dag/steps/run_star.py#L206
Please confirm :)

Reply:
resolved offline, changed to total+=2